packages/astro/src/core/routing/manifest/create.ts

```diff
@@ -520,6 +535,9 @@ export function createRouteManifest(
 
 		// we loop over the remaining routes and add them to the default locale
 		for (const route of setRoutes) {
+			if (route.type !== 'page') {
+				continue;
+			}
```

Comment:
Is this necessary?

Reply:
The i18n routing should be applied only to `page`. Although this check isn't needed here, I will move the logic in a better place

Reply:
Ah ok, I missed that from the RFC